frontend/scripts/react-components/shared/profile-selector/profile-selector.actions.js

```diff
@@ -109,10 +74,61 @@ export const setProfilesLoadingItems = loadingItems => ({
     loadingItems
   }
 });
+export const goToProfile = () => (dispatch, getState) => {
+  const { profileSelector, app } = getState();
+  const { contexts } = app;
+  const hasCompanies = !isEmpty(profileSelector.panels.companies.activeItems);
+  const profileType = hasCompanies ? 'actor' : 'place';
+  const profileSelection = hasCompanies
+    ? Object.values(profileSelector.panels.companies.activeItems)[0]
+    : Object.values(profileSelector.panels.sources.activeItems)[0];
+  const query = {
+    nodeId: profileSelection.id
+  };
+  const commodity = Object.values(profileSelector.panels.commodities.activeItems)[0];
```

Comment:
we dont need commodity

Reply:
This is to get the contextId it will change depending on the context

Reply:
I see, my bad